src/pages/Onboarding/index.tsx

```diff
@@ -1,5 +1,19 @@
+import { useState } from 'react';
+import SetZipCode from './SetZipCode';
+import UserInteraction from './UserInteraction';
+
 const OnboardingPage = () => {
-  return <div>OnboardingPage</div>;
+  const [isZipCodeSet, setIsZipCodeSet] = useState<Boolean>(false);
+
+  return (
+    <main className="inset-0 mx-5 mt-20 mb-[1.875rem] flex grow flex-col items-center justify-between overflow-hidden">
+      {isZipCodeSet === false ? (
+        <SetZipCode setIsZipCodeSet={setIsZipCodeSet} />
+      ) : (
+        <UserInteraction />
+      )}
```

Comment:
굉장히 사소한거지만 이렇게 바꿔도 좋을 것 같아요!

```suggestion
      {isZipCodeSet ? (
        <UserInteraction />
      ) : (
        <SetZipCode setIsZipCodeSet={setIsZipCodeSet} />
      )}
```

Reply:
수정했습니당